pkg/exporter/syslogexporter/README.md

```diff
@@ -1,29 +1,46 @@
 # Syslog Exporter
 
-Export/Forward syslog messages to a syslog server
-
 **Stability level**: Experimental
 
-This exporter can forward syslog messages to a third party [syslog server](https://www.rsyslog.com/)
+## About The Exporter
+
+The syslog exporter/forwarder supports sending messages to a remote syslog server.
+
+- This exporter/forwarder can forward syslog messages to a third party [syslog server](https://www.rsyslog.com/) using RFC5424 and RFC3164
+- It also supports sending syslog messages to the [Cloud Syslog Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-syslog-source/) configured on a Sumo Logic hosted collector using the RFC5424 format
+- The syslog exporter must be used with the [syslog_parser](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/syslog_parser.md)
+
+`Note` - Syslog over UDP doesn't support certificate verifiction (ca_certificate).
 
-This exporter/forwarder also supports sending syslog messages to the [Cloud Syslog](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-syslog-source/).
+## Configuration
+
+**The following are a few configuration options available to forward syslog messages**:
+
+- `endpoint` - syslog endpoint (FQDN or IP address)
+- `protocol` - tcp/udp
+- `port` - A syslog port
+- `format` - rfc5424/rfc3164
+  - `rfc5424` - Expects the syslog messages to be rfc5424 compliant
+  - `rfc3164` - Expects the syslog messages to be rfc3164 compliant
+- `additional_structured_data` [rfc5424 only] - Additional [structured data](https://www.rfc-editor.org/rfc/rfc5424#page-15) to specify in the syslog message
+- `ca_certificate` [tcp only] - A publicly verifiable server certificate (`note`: Self signed certificates are not supported in this version)
 
-Configuration is specified via the yaml in the following structure:
+Please refer to the yaml below to configure the syslog exporter:
 
 ```yaml
 extensions:
   file_storage/syslog:
-    directory: /tmp/otc
+    directory: /tmp/otc # Please ensure this directory exists
     timeout: 10s
```

Comment:
I don't think we need the storage configuration here. It's not really directly related to the exporter.

Reply:
Yes it is optional, so should we remove all references to the storage config from the readme?